docs/development/search.rst

```diff
@@ -65,10 +64,27 @@ with Django.
 
 Indexing
 ^^^^^^^^
+
 All the Sphinx documents are indexed into Elasticsearch after the build is successful.
 Currently, we do not index MkDocs documents to elasticsearch, but
 `any kind of help is welcome <https://github.com/readthedocs/readthedocs.org/issues/1088>`_.
 
+Troubelshooting
+^^^^^^^^^^^^^^^
+
+If you get an error like::
+
+   RequestError(400, 'search_phase_execution_exception', 'failed to create query: ...
+
+You can fix this by deleting the page index::
+
+   inv docker.compose 'exec search curl -X DELETE search:9200/page_index'
```

Comment:
I've been using `inv docker.manage 'search_index --delete'` and had the same effects. Maybe it's better to stay with django management commands.